### DISCUSSION TITLE
Is it a bug, type overload with map/apply

### DISCUSSION THREAD

**Discussion by PhanAlexandre** [microsoft/pyright#8985] [Q&A]
Hi.
I just enable TypeCheckingMode : "Basic"  for pyright, and it's show me correct line of code as errors.
Is it a bug or not?
From my understanding those type or class should be include by default in the Any type.
Here is some code with the errors
```
import pandas as pd
from pathlib import Path

df = pd.DataFrame({"a":["a","a","b","b","c"]})

df['b'] = df['a'].map(Path) #No overloads for "map" match the provided arguments
df['c'] = df['b'].apply(Path) #No overloads for "map" match the provided arguments
df['b'] = df['a'].map(list) #No overloads for "map" match the provided arguments
df['c'] = df['b'].apply(list)
df['b'] = df['a'].map(set) #No overloads for "map" match the provided arguments
df['c'] = df['b'].apply(set)
````

**Comment by erictraut** [comment]
You haven't specified which version of pandas you're using or whether you have the pandas-stubs stubs package installed in your environment. Those affect type checking.

This is unlikely to be a bug in pyright. If you have specific questions about pandas stubs, I recommend posting questions in [that project](https://github.com/pandas-dev/pandas-stubs). Its maintainers are knowledgeable about pandas and very responsive to typing-related questions.

**Reply by PhanAlexandre** [reply]
I have pandas 2.2.2
Installing solve some of those errors on 1 computer but not the other.
But I will check with the other project. Thanks you for the anwser
